dashboards/mysql/metadata.yaml

```diff
@@ -14,7 +14,7 @@ sample_dashboards:
     id: mysql-prometheus
     display_name: MySQL Prometheus Overview
     description: |-
-      This dashboard has charts displaying: Global Status Uptime (days), Available Connections, Open Files Available, Cache Hit Rate, MySQL Up, Global Status Connection Error Rate (5m), Slow Queries, Buffer Pool Cache Hit Ratio, Full Join, Full Range Join, Range Check, and Scan
+      This dashboard has charts displaying: Uptime (days), Available Connections, Open Files Available, Cache Hit Rate, MySQL Up, Connection Errors, Slow Queries, Buffer Pool Cache Hit Rate, Questions, Insert/Update/Delete Statements, Select Statements, Fsync Operations, and InnoDB Buffer Pool Read Requests vs Reads From Disk
```

Comment:
Just double-checking if this should match the README description exactly?

For example, the README correctly reflects that Uptime is now in minutes, not days.

Reply:
Thanks for catching this, it definitely should be minutes here.